lms/static/sass/views/_learner-profile.scss

```diff
@@ -307,8 +308,244 @@
             border: 2px dashed $link-color;
             .u-field-title,
             i {
-              color:  $link-color;
+                color: $link-color;
+            }
+        }
+    }
+
+    .badge-paging-header {
+        padding-top: $baseline;
+    }
+
+    .pagination {
```

Comment:
I'd agree with some of the remarks above - copying this code from Studio is understandable, but I would not like to have multiple versions of this UI styling pop up in the LMS side of things. I'm more comfortable if:

1. You abstract and move this set of styling to be its own partial or within a more universal components area within the LMS sass space
2. Or, less ideally, make a note note that this styling needs to be abstracted and should not be copied to other Sass files alongside a clean up story/issue to abstract this out.